### PR TITLE
Add basic support for MySQL replication

### PR DESCRIPTION
**Pull request by djmattyg007**
This supersedes #157, and has all of the following improvements

- Documentation links added where appropriate
- Removed unused function
- Support multiple slaves
- Fixed indentation in parts
- Removed `<code>` tags from keys
- Updated changelog
- Made new strings translatable

The one thing I don't yet have is a screenshot. I'm still working on that!

### CODE REVIEW

File: adminer/replication.inc.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	echo "<tr>";
+	echo "<th>" . h($key);
+	echo "<td>" . nbsp($val);
+}
+echo "</table>\n";
+
+$slave_replication_status = replication_status("SLAVE");
+if (!empty($slave_replication_status)) {
+	echo "<p><b>" . lang('Slave status') . "</b>" . doc_link(array("sql" => "show-slave-status.html")) . "\n";
+	foreach ($slave_replication_status as $slave) {
```
Comment (vrana) [inline adminer/replication.inc.php]: This should be `$slave_replication_status[0]`. Did you test it?

**Comment by vrana** [comment]
Merged with some changes and without the second commit. If you'll add the anchors to the MySQL doc then request it again.

**Comment by vrana** [comment]
Reverted based on https://sourceforge.net/p/adminer/discussion/960417/thread/5cbfab76/ and my own doubts.

**Comment by djmattyg007** [comment]
If you don't want this in the core, could you make it easier to add these sorts of things as plugins?

**Comment by vrana** [comment]
Probably not anytime soon.

**Pull request closed without merging**